app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split(" ")
+    file1 = command_list[1]
+    file2_path = command_list[-1]
```

Comment:
Code Style: Use descriptive and correct variable names. 'file1' and 'file2_path' could be more descriptive to indicate their purpose or content.